src/code/code_800F7260.c

```diff
@@ -1,40 +1,639 @@
 #include <ultra64.h>
 #include <global.h>
+#include <vt.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800F7260/func_800F7260.s")
+typedef struct {
+    /* 0x00 */ u16 sfxId;
+    /* 0x04 */ Vec3f* pos;
+    /* 0x08 */ u8 unk_8;
+    /* 0x0C */ u32* unk_C;
+    /* 0x10 */ u32* unk_10;
+    /* 0x14 */ u32* unk_14;
+} Struct_800F738C; // size = 0x18
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800F7260/func_800F72B8.s")
+typedef struct {
+    /* 0x00 */ f32 unk_0;
+    /* 0x04 */ f32 unk_4;
+    /* 0x08 */ f32 unk_8;
+    /* 0x0C */ u16 unk_C;
+} Struct_800F8EA0; // size = 0x10
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800F7260/func_800F731C.s")
+typedef struct {
+    u32 unk_0;
+    u8 unk_4;
+} Struct_800F7CEC;
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800F7260/Audio_PlaySoundGeneral.s")
+typedef struct {
+    u8 unk_0;
+    u16 unk_2;
+} Struct_8013331C;
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/code_800F7260/func_800F74E0.s")
+u8 D_80133340[4] = {
+    0x53,
+    0x45,
+    0x00,
+    0x00,
+};
 
+char D_80133344[] = VT_COL(RED, WHITE) "<INAGAKI CHECK> dist over! flag:%04X ptr:%08X pos:%f-%f-%f" VT_RST "\n";
+
+s32 D_8013338C = 0;
+
+UNK_TYPE D_80133390[] = {
+    0x53455120,
+    0x48000000,
+};
+
+UNK_TYPE D_80133398[] = {
+    0x20202020,
+    0x4C000000,
+};
+
+u8 D_801333A0 = 0;
+
+u8 D_801333A4 = 0;
+
+/**
+ * Array of pointers to arrays of SoundBankEntry of sizes: 9, 12, 22, 20, 8, 3, 5
+ *
+ * 0 : Player Bank          size 9
+ * 1 : Item Bank            size 12
+ * 2 : Environment Bank     size 22
+ * 3 : Enemy Bank           size 20
+ * 4 : System Bank          size 8
+ * 5 : Ocarina Bank         size 3
+ * 6 : Voice Bank           size 5
+ */
+SoundBankEntry* gSoundBanks[7] = {
+    D_8016BAD0, D_8016BC80, D_8016BEC0, D_8016C2E0, D_8016C6A0, D_8016C820, D_8016C8B0,
+};
+
+u8 sBankSizes[ARRAY_COUNT(gSoundBanks)] = {
+    9, 12, 22, 20, 8, 3, 5,
+};
```

Comment:
```suggestion
SoundBankEntry* gSoundBanks[7] = {
    D_8016BAD0, D_8016BC80, D_8016BEC0, D_8016C2E0, D_8016C6A0, D_8016C820, D_8016C8B0,
};

static u8 sBankSizes[ARRAY_COUNT(gSoundBanks)] = {
    ARRAY_COUNT(D_8016BAD0), ARRAY_COUNT(D_8016BC80), ARRAY_COUNT(D_8016BEC0), ARRAY_COUNT(D_8016C2E0), ARRAY_COUNT(D_8016C6A0), ARRAY_COUNT(D_8016C820), ARRAY_COUNT(D_8016C8B0),
};
```
`static` since you used the `s` prefix I guess this isn't used outside the file?

Reply:
Other code files such as `game.c` use the `s` prefix without making the variables `static` so I'm also reluctant to do that currently as we might be doing that at a later stage.

Reply:
yeah the `s` prefix doesnt mean `static` keyword, its two separate uses for the same word. can just treat it as local vs global